src/main/resources/locale_fr_FR.properties

```
@@ -0,0 +1,196 @@
+command.thankwords.sub.check.reference=Message de référence : [Sauter](%URL%)
+command.thankwords.sub.check.result=Donateur : %DONATOR%\nReceveur : %RECEIVER%.
+command.thankwords.sub.list=Liste de tous les motifs
+command.thankwords.sub.list.list=Les motifs suivants sont actifs :
+command.thankwords.sub.loadDefault=Charge les mots de remerciement par défaut pour une langue.
+command.thankwords.sub.loadDefault.added=Ajoute les mots de remerciement suivants :
+command.thankwords.sub.loadDefault.available=Les langues disponibles :
+command.thankwords.sub.remove=Supprimer un motif
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
+conversation.cancel=annuler
+conversation.canceled=Conversation annulée.
+conversation.exit=sortir
+conversation.finished=Conversation terminée.
+conversation.inProgress=Une conversation est déjà en cours dans ce canal. Tu peux quitter cette conversation en tapant $conversation.exit$ ou $conversation.cancel$.
+conversation.start=Démarre une nouvelle conversation.\NTu peux quitter la conversation à tout moment en tapant $conversation.exit$ ou $conversation.cancel$.
+error.commandNotFound=Commande non trouvée.
+error.invalidArguments=Arguments non valides
+error.invalidChannel=Canal non valide
+error.invalidMessage=Message non valide
+error.invalidNumber=Numéro non valide
+error.invalidRegex=Modèle regex non valide.
+error.invalidRole=Rôle non valide
+error.missingPermission=Il me manque la permission %PERM% pour fonctionner correctement.
+error.missingPermissionChannel=Ceci est dû au fait que la permission est prioritaire sur le paramètre du canal %CHANNEL%.
+error.missingPermissionGuild=Cela est dû aux paramètres de permission globaux.
+error.noValidChannel=Aucun canal valide n'a été fourni.
+error.notABoolean=%INPUT% n'est pas un booléen (true|false).
+error.notANumber=%INPUT% n'est pas un nombre.
+error.notYourEmbed=Ce n'est pas ton embed !
+error.prefixTooLong=Le préfixe ne peut comporter que 3 caractères.
+error.roleAccess=On ne peut pas interagir avec le rôle %ROLE%. Veuillez ajuster la hiérarchie.
+error.userNotFound=Impossible de trouver cet utilisateur.
+listener.messages.request.descr=Hey, tu viens de remercier quelqu'un, veux-tu remercier quelqu'un en particulier ? Si oui, clique sur le bouton ci-dessous.
+listener.messages.request.descrThank=Merci d'avoir donné de la réputation à quelqu'un. Tu peux donner %MORE% plus de réputation.
+listener.messages.request.title=Remerciements confirmation
+listener.reaction.confirmation=%DONOR% a donné de la réputation à %RECEIVER%.
+listener.reaction.removal=Réputation donnée par %DONOR% a été supprimée.
+message.welcome=Merci de m'avoir invité.\NTu peux démarrer une installation facile avec la commande /setup.
+messages.destruction=Ce message s'autodétruira dans %MIN% minute.
+thankType.answer=Réponse
+thankType.fuzzy=Fuzzy
+thankType.mention=Mention
+thankType.reaction=Réaction
+word.continue=Continue
```

Looks untranslated

DeepL suggestion:
```suggestion
word.continue=Continuer
```